tfhe/benches/shortint/casting.rs

```diff
@@ -1,16 +1,19 @@
 use crate::utilities::{write_to_json, OperatorType};
 use criterion::Criterion;
 use rayon::prelude::*;
+use tfhe::shortint::parameters::{
+    PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64, PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
```

Comment:
this replacement is not correct I believe, this should be the 2_2 gaussian (as before)

Reply:
Why? Can't we cast between Gaussian and TUniform?

Reply:
the ks parameters are specific to 1_1 and 2_2 IIRC

Reply:
Ok I'll put it back then